client/messages/cs.json

```
@@ -1144,12 +1177,17 @@
   "ciDaNU": "Automaticky zaznamenat první zobrazení stránky při načtení skriptu",
   "KzO2sB": "Sledování úvodního zobrazení stránky zapnuto",
   "FcSY6x": "Sledování úvodního zobrazení stránky vypnuto",
+  "5zGub0": "Sledování chyb",
```

Biome (2.4.12)

[error] The key 5zGub0 was already declared.

(lint/suspicious/noDuplicateObjectKeys)